doc/overview/ogc_overview.rst

```diff
@@ -8,39 +9,21 @@
-open interface standards and associated encoding standards, and also best
-practices, that enable developers to create information systems that can easily
-exchange “geospatial” information and instructions with other information
-systems. Requirements range from complex scheduling and control of Earth
-observation satellites to displaying simple map images on the Web and encoding
-location in just a few bytes for geo-tagging and messaging. A look at the `OGC
+The Open Geospatial Consortium (OGC) is an international consortium of more than 500 businesses, government agencies, research organizations, and universities driven to make geospatial (location) information and services FAIR - Findable, Accessible, Interoperable, and Reusable. OGC's member-driven consensus process creates royalty free, publicly available, open geospatial standards. Existing at the cutting edge, OGC actively analyzes and anticipates emerging tech trends, and runs an agile, collaborative Research and Development (R&D) lab - the OGC Innovation Program - that builds and tests innovative prototype solutions to members' use cases. OGC members together form a global forum of experts and communities that use location to connect people with technology and improve decision-making at all levels. OGC is committed to creating a sustainable future for us, our children, and future generations. A look at the `OGC
 Domain Working Groups <https://www.ogc.org/projects/groups/wg>`_
 shows the wide scope of current activity in the OGC.
 
-The OGC Standards Baseline consists of the |OGC| standards
-for interfaces, encodings,
+The OGC Standards Baseline consists of the |OGC| standards for interfaces, encodings,
 profiles, application schemas, and best practice documents. The |ORM|
 describes these
 standards and the relationships between them and related ISO standards. The ORM
 provides an overview of OGC standards and serves as a useful resource for
 defining architectures for specific applications.
 
-In developing a Web services application using OGC standards (and in learning
-about the relationships between OGC standards) it helps to think of publish,
-find and bind as the key functions for applications in a Web services
-environment.
+In 2020, the `OGC API <https://ogcapi.ogc.org/>`_ family of standards started to be developed to make it easy for anyone to provide geospatial data to the web. These standards build upon the legacy of the OGC Web Service standards (WMS, WFS, WCS, WPS, etc.), but define resource-centric APIs that take advantage of modern web development practices. This web page provides information on these standards in a consolidated location. These standards are being constructed as "building blocks" that can be used to assemble novel APIs for web access to geospatial content. The building blocks are defined not only by the requirements of the specific standards, but also through interoperability prototyping and testing in OGC's Innovation Program.
```

Comment:
You can use the automatic links with expanded names that are used on the projects documentation and are defined here:
https://github.com/OSGeo/OSGeoLive-doc/blob/master/doc/conf.py#L320
as follows:
```
standards (|WMS|, |WFS|, ....
```
Now with the expanded names you can use a list
```
* |WMS|
* |WFS|
```


And now that you are at it, maybe you can double check acronyms that are missing or wrong names for the links in the `conf.py` file.
It would be better to make the changes of `conf.py` in this same PR otherwise we can have an ordering problem on how to merge the PR's like in the other ones where we can not merge INSPIRE because it depend on the inclusion or not of re3gistry on the disk  and only after re3gistry is included we can merge the `conf.py` PR you already made, before merging INSPIRE.

Side note:
Its difficult to pinpoint exactly the line where exactly because your lines are too long
Consider how sphinx works:

```
line1
line2
line3
line4

line5
line6
line7

```
line1 to line4 will belong to a first paragraph
line5 to line7 will belong to a second paragraph
Because they are separated by a blank space.

Reply:
@df-git I made my review as a comment, if you think that without the suggestion on the comment its ok, and no need to change things here or on conf.py please let me know so that I approve from my part (I don't check grammar or spelling, I just check that documentation generation is built properly)

Reply:
Dear @cvvergara,
Thank you for the suggestion. I think you can go on like it is now.